### DISCUSSION TITLE
Flutter

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#34726] [Other Feature Feedback, Questions, & Ideas]
Hey guys!

Okay, so I am trying something different, and I a totally new in this. What is Flutter and why do we use that? I have been asked a number of questions in my college about whether you do flutter or not. Whether you have any experience in Flutter or not? And I was like damn I should know this!

So please help me, guys. Provide me with some documentation so that I can learn well.

Thank you so much, everyone!
Happy Learning...✌🏻

**Comment by jfullstackdev** [comment]
it says here it is a Software Development Kit

![image](https://user-images.githubusercontent.com/47092464/193393882-cf5fdb32-8636-431b-abc6-fe585f8c04c1.png)

it's a collection of software development tools in one installable package, so it will simplify the installation of everything

if you are familiar with Java, there is the Java Development Kit JDK, so you can program and run Java programs because it has its own environment, say, when you install it on Windows, 

you may want to check this out too

https://www.geeksforgeeks.org/differences-jdk-jre-jvm/

you know, in actual development, there will always be conflict when there are developers working in the same project and they are using different setup, the other user will say, this is working on my end, the other one will say, no it's not, so you need to have the same environment

**Comment by fal3n-4ngel** [comment]
![Screenshot 2022-11-12 080002](https://user-images.githubusercontent.com/79042374/201452506-0b15e190-6329-4885-a6a9-2ab9bb441b0b.png)

### Flutter is a Google's portable UI toolkit used for developing android/ios /windows applications
Everything you wanna know is already provided in the [website](https://flutter.dev/) . Flutter [official documentations](https://docs.flutter.dev/) are literally the best.

**Reply by farhansadikgalib** [reply]
It's really awesome.

**Comment by Debajyoti14** [comment]
![image](https://user-images.githubusercontent.com/91759192/208771268-f2dc2139-5b2d-41f6-bb9e-844b08119c83.png)
### Learn from the best 😅✌️

**Comment by jarvis-vision4** [comment]
Flutter is modern cross platform development dart framework.

**Comment by soycodigorafa** [comment]
Flutter is life, Flutter is love
